shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -10,78 +10,123 @@
 
 #include "FlutterBinaryMessenger.h"
 #include "FlutterDartProject.h"
+#include "FlutterEngine.h"
 #include "FlutterMacros.h"
 #include "FlutterPlugin.h"
 #include "FlutterTexture.h"
 
+@class FlutterEngine;
+
+/**
+ * A `UIViewController` implementation for Flutter views.
+ *
+ * Dart execution, channel communication, texture registration, and plugin registration
+ * are all handled by `FlutterEngine`.  Calls on this class to those members all proxy
+ * through to the `FlutterEngine` that manages this FlutterViewController.
```

Comment:
'manages' seems to be too strong of a word. The owner of this FlutterViewController 'manages' this FlutterViewController.

Reply:
Sure. I assume it's still possible to make a FlutterViewController just via initWithNibName:bundle:? What happens when no FlutterEngine is attached?